ui/admin/tests/acceptance/roles/scopes-test.js

```diff
@@ -162,13 +218,134 @@ module('Acceptance | roles | scopes', function (hooks) {
         },
       );
     });
-    instances.role.update({ grant_scope_ids: [] });
-    await visit(urls.role);
+    instances.orgRole.update({ grant_scope_ids: [] });
+    await visit(urls.orgRole);
 
     await click(MANAGE_SCOPES_SELECTOR);
     await click(SCOPE_TOGGLE_SELECTOR(GRANT_SCOPE_THIS));
     await click(SAVE_BTN_SELECTOR);
 
     assert.dom(TOAST_SELECTOR).isVisible();
   });
+
+  test('user can select and save custom scopes to add on manage custom scopes page', async function (assert) {
```

Comment:
Non-blocking:
I expected to see acceptance tests for the manage custom scopes route in it's own test file. However, thinking about this a bit, I wonder if we should split these tests by scope level. This whole workflow is different for global, org, and project level scopes. Might be worth splitting it up that way to better test the workflow in it's entirety.

Reply:
that is a very good point, since this pr is already a bit chunky ill include a refactor for this in the next PR! Thank you!